segmentstore/server/host/src/test/java/io/pravega/segmentstore/server/host/ServiceStarterTest.java

```diff
@@ -54,16 +67,25 @@ public void stopZookeeper() throws IOException {
      */
     @Test
     public void testCuratorClientCreation() throws Exception {
-        ServiceBuilderConfig.Builder configBuilder = ServiceBuilderConfig
-                .builder()
-                .include(ServiceConfig.builder()
-                        .with(ServiceConfig.CONTAINER_COUNT, 1)
-                        .with(ServiceConfig.ZK_URL, zkUrl));
-        @Cleanup("shutdown")
-        ServiceStarter serviceStarter = new ServiceStarter(configBuilder.build());
-        @Cleanup
         CuratorFramework zkClient = serviceStarter.createZKClient();
```

Comment:
Do not remove `@Cleanup` from zkClient. That ensures it closes it regardless of whether your test succeeds or fails.